site/docsource/Get_started.adoc

```diff
@@ -2,22 +2,37 @@
 
 ### First example
 
-* Create a directory called `hello` and create `package.json`
-  as below:
+* Create a directory called `hello`
+* Init the project as a NPM package and fill-in the details
+----
+npm init
+----
+* Add bucklescript as a dev dependency
+-----
+npm install --save-dev bs-platform
+-----
+* In the package.json, remove the `test` script and add a `build` script. You're file should look like:
 +
 [source,js]
 .package.json
 ----
 {
-    "dependencies": {
-        "bs-platform": "0.9.3" // <1>
+  {
```

Comment:
I like the idea to use `npm init`, however, the original package.json is *minimal* so that people have a clear idea what's inside. how about adding a link to `npm init` to tell them the more idiomatic way /